tests/_support/Step/Acceptance/Administrator/ContentSteps.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace Step\Acceptance\Administrator;
+
+class ContentSteps extends \AcceptanceTester
+{
+	/**
+	 * @Given There is a Add Content link
+	 */
+	public function thereIsAAddContentLink()
+	{
+		$I = $this;
+		$I->amOnPage('administrator/index.php?option=com_content&view=articles');
+		$I->clickToolbarButton('New');
+	}
+
+	/**
+	 * @When I fill mandatory fields for creating article
+	 */
+	public function iFillMandatoryFieldsForCreatingArticle(\Behat\Gherkin\Node\TableNode $fields)
+	{
+		$I = $this;
+		// iterate over all rows
+		foreach ($fields->getRows() as $index => $row) {
+			if ($index === 0) { // first row to define fields
+				$keys = $row;
+				continue;
+			}
+			else
```

Comment:
This code could be improved ;)

...
}
elseif ($row[0] == 'title') 
{
...
}
elseif ($row[0] == 'content') 
{
...
}

Instead of the else.

$row[0] is also not good readable, it would be nice when you could do the following above:

$key = $row[0];
$value = $row[1];

That would improve readability